src/App.js

```diff
@@ -0,0 +1,65 @@
+// App.js
+
+import React, { useEffect } from 'react';
+import {
+  BrowserRouter, Route, Routes, Navigate,
+} from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import SignUpForm from './components/sessions/Signup';
+import LoginForm from './components/sessions/Login';
+import Profile from './components/Profile';
+import CarDetails from './components/CarDetails';
+import DeleteCar from './components/car/DeleteCar';
+import Sidebar from './components/navigation/Sidebar';
+import Reserve from './components/reservation/CarReservationForm';
+import UserReservations from './components/reservation/UserReservations';
+import CreateCar from './components/car/CreateCarForm';
+import { loginSuccess } from './redux/reducer/authSlice';
+import './App.css';
+
+const App = () => {
```

Comment:
- [ ] When I press the logout button , I just get this screen:

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/96a4c499-2b06-4e0c-8c0f-474fd63ad11b)

Kindly, make it so when your users logout, they return to the login screen instead of getting a blank screen.  

That way, your UX will be improved. 👍